pom.xml

```diff
@@ -43,7 +43,7 @@
     <tika.version>1.22</tika.version>
     <jackson.version>2.12.1</jackson.version>
     <jacksondatabind.version>2.12.1</jacksondatabind.version>
-    <httpclient.version>4.5.13</httpclient.version>
+    <httpclient.version>4.5.5</httpclient.version>
```

Comment:
I traced that @sfc-gh-hthirividi upgrade to 4.5.13 four months ago to fix the issue. However, I don't have much time to go through all the discussions. I would suggest to at least put some comments here to describe why use 4.5.5 and what is the rule we should follow in the future to upgrade the version. Also, since we lose the benefits of 4.5.13, it deserves an explanation.